packages/core/src/components/header/header-dropdown/header-dropdown.tsx

```diff
@@ -43,6 +43,13 @@ export class TdsHeaderDropdown {
     this.open = !this.open;
   }
 
+  handleSlottedItemClick = (event: MouseEvent) => {
+    const eventSource = (event.target as HTMLElement).tagName.toLowerCase();
+    if (eventSource === 'a' || eventSource === 'button') {
```

Comment:
We could rewrite it like this also:

```
if (['a', 'button'].includes(eventSource)) {
    this.open = false;
  }
```

Potentially more readable if we add more tag names going forward?

Reply:
Great idea @timrombergjakobsson ! Thanks for suggestion ;)